src/components/course_info.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
```

Comment:
haven't used react in a long while - is the required?

Reply:
seems to be unused?

Reply:
probably not... On it :)

Reply:
This has been removed!